src/platforms/javascript/common/session-replay/understanding-sessions.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: Understanding Sessions
+sidebar_order: 4300
+notSupported:
+  - javascript.cordova
+  - javascript.electron
```

Comment:
We support Electron:

```suggestion
```

Reply:
Ah, need to remove this from the other docs as well then. will do!